crates/shared/src/price_estimation/instrumented.rs

```diff
@@ -136,6 +132,6 @@ mod tests {
             .price_estimation_times
             .with_label_values(&["foo"])
             .get_sample_count();
-        assert_eq!(observed, 1);
+        assert_eq!(observed, 2);
```

Comment:
This value changed because we now have 2 separate calls instead of 1 call with 2 queries.